src/main/resources/reference.conf

```diff
@@ -156,4 +156,46 @@ cassandra-snapshot-store {
       parallelism-max = 8
     }
   }
+}
+
+cassandra-query-journal {
```

Comment:
The intention here was to keep the journals completely separated for clients to choose what they need.